Getting Started/Installation.rst

```diff
@@ -3,6 +3,14 @@ Installation
 
 osTicket comes with its own web-based installer to help guide you through the installation process without the frustration. While the installer provides step by step guide during the installation process, it's important and helpful to have general knowledge about Web servers, PHP and MySQL.
 
+Centos:
```

Comment:
I would suggest the following:
```
**CentOS 7**
```

This way the title is bold/more noticeable and you specify which CentOS version you're using.

Reply:
Done!